### PR TITLE
feat(query): added 'Operation Using Password Flow' for swagger

### PR DESCRIPTION
**Pull request by rafaela-soares**
Signed-off-by: Rafaela Soares <rafaela.soares@checkmarx.com>

**Proposed Changes**
- Added 'Operation Using Password Flow' for swagger

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/openAPI/2.0/operation_using_password_flow/query.rego [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) == "2.0"
+
+	doc.paths[p][oper].security[n][key]
+	doc.securityDefinitions[key].flow == "password"
+	doc.securityDefinitions[key].type == "oauth2"
```
Comment (felipe-avelar) [inline assets/queries/openAPI/2.0/operation_using_password_flow/query.rego:11]: Based on [this comment](https://github.com/Checkmarx/kics/pull/3664#discussion_r652498677), I think `doc.securityDefinitions[key].type == "oauth2"` is not necessary

File: assets/queries/openAPI/2.0/security_definitions_using_password_flow/query.rego [hunk 2]
```diff
@@ -6,14 +6,14 @@ CxPolicy[result] {
 	doc := input.document[i]
 	openapi_lib.check_openapi(doc) == "2.0"
 
-	secDef := doc.securityDefinitions[key]
-	secDef.flow == "password"
+	doc.securityDefinitions[key].flow == "password"
+	doc.securityDefinitions[key].type == "oauth2"
```
Comment (felipe-avelar) [inline assets/queries/openAPI/2.0/security_definitions_using_password_flow/query.rego:10]: Based on [this comment](https://github.com/Checkmarx/kics/pull/3664#discussion_r652498677), I think `doc.securityDefinitions[key].type == "oauth2"` is not necessary

**Review by felipe-avelar — CHANGES_REQUESTED** [review]
Please, check comments

**Review by felipe-avelar — APPROVED** [review]
LGTM

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**